metabase/metabase
# Remove `SaveQuestionModal.module.css` and replace with general CSS modules

### Description

Removes the unused classes in `SaveQuestionModal.module.css` and replaces the one used class with a core CSS module. It looks like the animation isn't working on the save question modal, which we can change in the future, but for now we can just remove these classes and the component

### How to verify

Create and save a question. The transition and the form area should not overflow outside of its container.
